feat: add fortune telling app (#62167)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-fortune-teller-app/68ed42f744fc8aa6d479e926.md

---
id: 68ed42f744fc8aa6d479e926
title: Step 18
challengeType: 0
dashedName: step-18
---

# --description--

Now add the remaining properties to the elements object. Make sure each property uses the `HTMLElement` type.
`multipleCard`, `title`, `newReadingBtn`, `fortuneContainer`, `fortuneDescription`, `headerTitle`, `subTitle`,

> `cardTitle`, `description`, `text`

Reviewer: ```suggestion
Now add the remaining properties to the `elements` object. Make sure each property uses the `HTMLElement` type: `multipleCard`, `title`, `newReadingBtn`, `fortuneContainer`, `fortuneDescription`, `headerTitle`, `subTitle`, `cardTitle`, `description`, and `text`.
```

> 

Reviewer: ```suggestion
```

# --hints--

> 

Reviewer: ```suggestion
```

Inside elements method you should have `multipleCard` property assigned to `HTMLElement`.

```js
assert.match(code, /multipleCard\s*:\s*HTMLElement\s*;/);
```

> Inside elements method you should have `title` property assigned to `HTMLElement`.

Reviewer: ```suggestion
Inside the `elements` property, the `title` property should be assigned the type `HTMLElement`.
```

> Inside elements method you should have `title` property assigned to `HTMLElement`.

Reviewer: ```suggestion
You should have the `title` property inside the `elements` property assigned the type `HTMLElement`.
```

```js
assert.match(code, /title\s*:\s*HTMLElement\s*;/);
```

Inside elements method you should have `newReadingBtn` property assigned to `HTMLElement`.

```js
assert.match(code, /newReadingBtn\s*:\s*HTMLElement\s*;/);
```

Inside elements method you should have `fortuneContainer` property assigned to `HTMLElement`.

```js
assert.match(code, /fortuneContainer\s*:\s*HTMLElement\s*;/);
```

Inside elements method you should have `fortuneDescription` property assigned to `HTMLElement`

```js
assert.match(code, /fortuneDescription\s*:\s*HTMLElement\s*;/);
```

> Inside elements method you should have `headerTitle` property assigned to `HTMLElement`.

Reviewer: ```suggestion
Inside the `elements` property, the `headerTitle` property should be assigned the type `HTMLElement`.
```

```js
assert.match(code, /headerTitle\s*:\s*HTMLElement\s*;/);
```

> Inside elements method you should have `subTitle` property assigned to `HTMLElement`.

Reviewer: ```suggestion
Inside the `elements` property, the `subTitle` property should be assigned the type `HTMLElement`.
```

```js
assert.match(code, /subTitle\s*:\s*HTMLElement\s*;/);
```

> Inside elements method you should have `cardTitle` property assigned to `HTMLElement`.

Reviewer: ```suggestion
Inside the `elements` property, the `cardTitle` property should be assigned the type `HTMLElement`.
```

```js
assert.match(code, /cardTitle\s*:\s*HTMLElement\s*;/);
```

Inside elements method you should have `description` property assigned to HTMLElement.

```js
assert.match(code, /description\s*:\s*HTMLElement\s*;/);
```

> Inside elements method you should have `text` property assigned to `HTMLElement`.

Reviewer: ```suggestion
Inside the `elements` property, the `text` property should be assigned the type `HTMLElement`.
```
